HGustavs/LenaSYS
# Pressing ESC button does not turn element to "none" in markdown editor

It was likely made this way to avoid the event firing for the window underneath, which would move you back to the sectioned page.
Now the one underneath has been given priority on checking for events and the eventlistener on the popup window itself is using the same function as the other buttons.

**testing:**

- go to any course with a codeexample (webbprogrammering for example)
- go to a codeexample
- click the edit button:
![image](https://github.com/HGustavs/LenaSYS/assets/129263158/5549e0d0-ca93-4709-acd6-566aec88fec1)
- press escape and check if it sets previewWindow loginBox to display:none
